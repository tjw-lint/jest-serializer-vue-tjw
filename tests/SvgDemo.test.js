const helpers = require('./helpers.js');

import { mount } from '@vue/test-utils';
import SvgDemo from './components/SvgDemo.vue';

describe('SvgDemo.vue', () => {
  test('SVG renders properly', () => {
    helpers.mockSettings({});
    const wrapper = mount(SvgDemo);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
