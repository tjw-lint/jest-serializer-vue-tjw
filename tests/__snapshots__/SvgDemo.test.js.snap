// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgDemo.vue SVG renders properly 1`] = `
<div>
  <svg height="156">
    <desc>Example Triangle</desc>
    <path d="M 10,150 L 70,10 L 130,150 z" fill="#D5D8CB" stroke="#ECDCC6" stroke-width="6" />
  </svg>
  <svg height="80px" viewBox="0 0 400 400" width="100%" xmlns="http://www.w3.org/2000/svg">
    <path d="M 100 100 L 300 100 L 200 300 z" fill="LightBlue" stroke="Blue" stroke-width="3" />
  </svg>
</div>
`;
